C7/MainMenu.cs

```diff
@@ -61,14 +61,20 @@ private void DisplayTitleScreen()
 		}
 		catch(Exception ex)
 		{
-			GD.Print("Could not set up the main menu: ", ex);
+			GD.Print("Could not set up the main menu", ex);
+			GetNode<Label>("CanvasLayer/Label").Visible = true;
+			GetNode<ColorRect>("CanvasLayer/ColorRect").Visible = true;
```

Comment:
I made these hidden by default, as part of the refactor, so they now have to become visible when we can't load the PCX files for things such as button graphics.